use Unicode glyphs or SVG data URLs instead of bitmaps, tidy the CSS and fix IVY-450

twogee opened the pull request:
I put this up for review in case I missed some page that used the removed icons.

twogee committed:
use Unicode glyphs or SVG data URLs instead of bitmaps

jaikiran commented on build-release.xml:
```diff
@@ -179,11 +180,11 @@
     <target name="init-asciidoctor" depends="init-asciidoctor-extensions" unless="skip.doc">
-        <ivy:cachepath pathid="asciidoctor.path" organisation="org.asciidoctor" module="asciidoctor-ant" revision="1.5.4" inline="true"/>
+        <ivy:cachepath pathid="asciidoctor.path" organisation="org.asciidoctor" module="asciidoctor-ant" revision="1.5.3" conf="default" inline="true"/>
         <taskdef uri="antlib:org.asciidoctor.ant" resource="org/asciidoctor/ant/antlib.xml" classpathref="asciidoctor.path">
```
Instead of downgrading the version, I think we should try and figure out whatever the issue is with 1.5.4

twogee replied:
Very well, here's the output from 1.5.4:

```
.../ant-ivy/build-release.xml:202: org.jruby.exceptions.RaiseException: (LoadError) no such file to load -- asciidoctor
	at org.jruby.RubyKernel.require(org/jruby/RubyKernel.java:1040)
	at RUBY.require(/Users/gintas/.ivy2/cache/org.asciidoctor/asciidoctor-ant/jars/asciidoctor-ant-1.5.4.jar!/META-INF/jruby.home/lib/ruby/shared/rubygems/core_ext/kernel_require.rb:54)
	at RUBY.(root)(<script>:9)
```

twogee committed:
IVY-450 highlight current page in menu

twogee commented:
I changed back the version of asciidoctor-ant to 1.5.4 and added a stab at IVY-450. The current description does not indicate what should happen when the parent is collapsed, so I assume that one may as well use breadcrumbs rather than visual cues from the menu.

twogee commented:
Looks like I've beaten time to first response record again: 90000+ hours 😃

maartenc commented:
I've merged a modified version of your highlighting code + css cleanup.
I didn't do the SVG changes since I like the old shapes more. Maybe if you create new SVGs that match better match the old shapes, I could merge them.
